docs/tutorials/routing-by-namespace.rst

```diff
@@ -26,7 +23,7 @@ Let's assume you have an existing Slack sink:
         slack_channel: app-notifications
         api_key: secret-key
 
-First, duplicate duplicate your sink. You need a unique sink for each channel you want to notify in:
+By default, the sink will receive notifications for all namespaces. Before we fix that, let's duplicate the sink, changing only the Slack channel:
```

Comment:
"Before we fix that" - We are adding a feature and right now it is working as it's supposed to. So maybe something like "Before we modify that" or  "Before we change that" would be better?